### PR TITLE
Add gmf module in ngeo-based interface

### PR DESCRIPTION
**Pull request by asaunier**

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/+package+/templates/ngeo.html_tmpl [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <title translate>The {{project}} Web GIS</title>
   <head>
   <body>
-    <app-map app-map-map="::mainCtrl.map"></app-map>
+    <gmf-map gmf-map-map="mainCtrl.map"></gmf-map>
```
Comment (fredj) [inline c2cgeoportal/scaffolds/create/+package+/templates/ngeo.html_tmpl]: Any reason to remove the one-time binding ?

Comment (fredj) [inline c2cgeoportal/scaffolds/create/+package+/templates/ngeo.html_tmpl]: See https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/map.js#L7

**Comment by fredj** [comment]
Thanks for any review

**Comment by sbrunner** [comment]
Looks good for me :-)

**Comment by ger-benjamin** [comment]
+1

**Comment by fredj** [comment]
thanks for the review

**Pull request merged**